build.sh

```diff
@@ -1,5 +1,17 @@
+mkdir -p bin
```

Comment:
Might want a `#!/usr/bin/env bash`, and `set -e` in this file too to ensure the correct interpreter is used and we fail fast.

```suggestion
#!/usr/bin/env bash

set -e

mkdir -p bin
```